src/python/WMCore/REST/Auth.py

```diff
@@ -25,6 +25,9 @@ def user_info_from_headers(key, verbose=False):
 
     # Reject if request was not authenticated.
     if 'cms-auth-status' not in headers:
+        if headers.get('Remote-Addr', '') == '127.0.0.1':
+            # since we are using localhost access we can skip authentication
```

Comment:
Valentin, I would suggest to take the same approach as the block below. So, please add a log here to identify that unauthenticated access has been performed (e.g.):
```
log("DEBUG: non-authenticated request performed")
```

Reply:
done